redwood-tooling-schema/src/main/kotlin/app/cash/redwood/tooling/schema/schemaParserFir.kt

```diff
@@ -374,19 +450,224 @@ private fun FirContext.parseLayoutModifier(
     )
   }
 
+  val deprecation = findDeprecationAnnotation(firClass.annotations)
+    ?.toDeprecation()
+
   return ParsedProtocolLayoutModifier(
     tag = tag,
-    scopes = scopes,
+    scopes = annotation.scopes,
     type = memberType,
-    deprecation = TODO(),
+    deprecation = deprecation,
     properties = properties,
   )
 }
 
```

Comment:
Everything below this comment is the new part, although a lot of it was moved from above.